Added behavior for deactivated users (#867)
spryker/spryker-docs: docs/marketplace/user/features/202108.0/marketplace-merchant-feature-overview/marketplace-merchant-feature-overview.md

| Active | N/A | &check; |
| Inactive | N/A | ✗ |

> For details about how to change the merchant statuses and activate and deactivate merchants in the Back Office, see [approving and denying merchants](/docs/marketplace/user/back-office-user-guides/{{page.version}}/marketplace/merchants/managing-merchants.html#approving-and-denying-merchants) and [activating and deactivating merchants](/docs/marketplace/user/back-office-user-guides/{{page.version}}/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).

Reviewer: ```suggestion
For details about how to change the merchant statuses and activate or deactivate merchants in the Back Office, see [approving and denying merchants](/docs/marketplace/user/back-office-user-guides/{{page.version}}/marketplace/merchants/managing-merchants.html#approving-and-denying-merchants) and [activating and deactivating merchants](/docs/marketplace/user/back-office-user-guides/{{page.version}}/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).
```

{% info_block infoBox "Info" %}

> Whenever a merchant is denied, all its users (/docs/marketplace/user/features/{{page.version}}/marketplace-merchant-feature-overview/merchant-users-overview.html) get deactivated. Upon re-approval, the merchant users must be activated manually and independently one by one.

Reviewer: ```suggestion
Whenever a merchant is denied, all the users (/docs/marketplace/user/features/{{page.version}}/marketplace-merchant-feature-overview/merchant-users-overview.html) of this merchant get deactivated. Upon re-approval, the merchant users must be manually activated one by one.
```
